lib/rodauth/features/jwt_refresh.rb

```diff
@@ -32,7 +32,6 @@ module Rodauth
     )
 
     route do |r|
-      @jwt_refresh_route = true
```

Comment:
Please keep this for backwards compatibility, maybe with a `# RODAUTH3` comment added to mark it as something we can drop in the next major version.